ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```diff
@@ -415,6 +518,58 @@ private static void SetOverflowVisible(TFrameworkElement element)
             element.Clip = null;
         }
 
+        private static async Task<JObject> GetDataTransferInfo(DataPackageView data, bool drop = false)
+        {
+            var files = new JArray();
+            var items = new JArray();
+            var types = new JArray();
+
+            if (data.Contains(StandardDataFormats.StorageItems))
+            {
+                foreach (var item in await data.GetStorageItemsAsync())
+                {
+                    try
+                    {
+                        var file = item as StorageFile;
+
+                        if (file != null)
+                        {
+                            var props = await file.GetBasicPropertiesAsync();
+                            var type = file.ContentType;
+                            var path = drop ? FutureAccessList.Add(file).ToString() : "";
```

Comment:
>.ToString() [](start = 72, length = 11)

Why do we need to call `ToString()`?

Reply:
Must be a leftover from the previous iteration.